pkg/core/image_manifest.go

```diff
@@ -29,7 +29,7 @@ const imageManifestVersion_0_1 = "0.1"
 // ImageManifest defines the image names found in YAML files of system components.
 type ImageManifest struct {
 	ManifestVersion string `json:"manifestVersion"`
-	Images          []string
+	Images          map[string]string
```

Comment:
Could it be useful to introduce a type alias for image coordinates and image sha, or do you think it's gonna be cumbersome?

Reply:
No, I think that could disambiguate the types nicely. Thanks! I suggest `imageName` and `imageDigest` for type names.